File: leetcode/1619-mean-of-array-after-removing-some-elements/trimMean.ts
function trimMean(arr: number[]): number {
  arr.sort((a, b) => a - b);
  const length = arr.length;
  const start = length * 0.05;
  const end = length * 0.95;

  let sum = 0;
  for (let index = start; index < end; index++) {
    sum = sum + arr[index]!;
  }

  return sum / (length * 0.9);
}

export default trimMean;

File: leetcode/1619-mean-of-array-after-removing-some-elements/trimMean.go
package leetcode

import "sort"

func trimMean(arr []int) float64 {
	sort.Ints(arr)
	var length = len(arr)
	var sum = 0
	for _, num := range arr[length/20 : length/20*19] {
		sum = sum + num
	}

	return float64(sum) / (float64(length) * 0.9)
}
